src/main/java/winstone/accesslog/SimpleAccessLogger.java

```diff
@@ -78,8 +81,12 @@ public SimpleAccessLogger(String webAppName, Map<String, String> startupArgs)
         } catch (Exception ex) {
             Logger.logDirectMessage(Logger.WARNING, null, "Failed to mkdirs " + parentFile.getAbsolutePath(), ex);
         }
-        this.outStream = new FileOutputStream(file, true);
-        this.outWriter = new PrintWriter(this.outStream, true);
+        try {
+            this.outStream = Files.newOutputStream(file.toPath(), StandardOpenOption.CREATE, StandardOpenOption.APPEND);
+        } catch (InvalidPathException e) {
+            throw new IOException(e);
+        }
+        this.outWriter = new PrintWriter(new OutputStreamWriter(this.outStream, StandardCharsets.UTF_8), true);
```

Comment:
Technically a change in behavior, but I think it makes the most sense for this log to be in UTF-8.

Reply:
as it's a breaking change the PR should have the appropriate label...

Reply:
> as it's a breaking change the PR should have the appropriate label...

I updated the label and the changelog to mention this change.